frontend/javascripts/oxalis/model_initialization.ts

```diff
@@ -281,6 +281,7 @@ function initializeTracing(_annotation: APIAnnotation, serverTracings: Array<Ser
         ...annotation,
         restrictions: {
           ...annotation.restrictions,
+          // todo: is this just an alias? can this be removed?
           allowSave: annotation.restrictions.allowUpdate,
```

Comment:
@daniel-wer Do you know why the front-end unpacks the `allowUpdate` restriction to `allowSave`? Is there a difference between these two?

Reply:
Yes there are differences. In line 277 (a couple of lines above) the sandbox case is handled, where `allowUpdate` is true, but `allowSave` is false.
`allowSave` is also used to disable saving when the `Disable Saving` button from the dropdown next to the save button is pressed. In that case, updates should still be allowed which also requires this distinction.

Reply:
Ah, great. Thanks for clearing up my confusion!